subsys/bluetooth/controller/ll_sw/ull_llcp_conn_upd.c

```diff
@@ -495,12 +537,23 @@ static void rp_cu_conn_param_req_ntf(struct ll_conn *conn, struct proc_ctx *ctx)
 
 static void rp_cu_complete(struct ll_conn *conn, struct proc_ctx *ctx, uint8_t evt, void *param)
 {
-	if (!ntf_alloc_is_available()) {
-		ctx->state = RP_CU_STATE_WAIT_NTF;
-	} else {
-		rp_cu_ntf(conn, ctx);
+	/*
```

Comment:
I'd prefer refactoring to not have complete-case code duplicated, but rather have a guard on the ntf to skip if not needed? Perhaps also change the prams_changed flag into a ntf flag - allowing the incorporation of the check for ctx->proc == PROC_CONN_UPDATE

Reply:
As above, the check for ctx->proc == PROC_CONN_UPDATE does not apply and will be removed.
Agree with the duplicated code stuff - but this must come be later...